packages/picker/tsconfig.json

```diff
@@ -7,13 +7,8 @@
     "include": ["*.ts", "src/*.ts", "sync/*.ts"],
     "exclude": ["test/*.ts", "stories/*.ts"],
     "references": [
-        { "path": "../button" },
-        { "path": "../icon" },
-        { "path": "../icons-ui" },
-        { "path": "../icons-workflow" },
+        { "path": "../base" },
```

Comment:
Here (the entries in the old version that are not in the new one) is the conversion where we missed these references that were putting a kink in the CI processes.

Reply:
I will add them back!